# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/generic/crashtests/1562105.html
# Commit: f770099a60cb
# Full Hash: f770099a60cb34e14a40ee8170807972398aa219
# Author: Hiroyuki Ikezoe <hikezoe@mozilla.com>
# Date: 2019-06-29 09:58:05
# Description:
#   Bug 1562105 - Bail out if the style frame for the scrollable frame is null in ScrollFrameHelper::IsSmoothScroll. r=botond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D36303
# ==============================================================================

diff -r b8d39cd984a2 -r f770099a60cb layout/generic/crashtests/1562105.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/generic/crashtests/1562105.html	Fri Jun 28 16:45:54 2019 +0000
@@ -0,0 +1,6 @@
+<script>
+document.addEventListener("DOMContentLoaded", function(){
+  document.documentElement.hidden = 'true'
+  window.scrollBy(9, 13)
+})
+</script>